shopfloor/actions/message.py

```diff
@@ -72,6 +78,15 @@ def package_not_found(self):
             "body": _("This package does not exist anymore."),
         }
 
+    def package_different_change(self):
+        return {
+            "message_type": "warning",
+            "body": _(
+                "You scanned a different package with the same product, "
+                "do you want to change pack ? Scan it again to confirm"
```

Comment:
```suggestion
                "do you want to change pack? Scan it again to confirm"
```
space is only for French I guess :smile:

Reply:
Space after the question mark in French, yes.
Fixed